hubstorage/client.py

```diff
@@ -40,7 +40,7 @@ def _hc_retry_on_exception(err):
 class HubstorageClient(object):
 
     DEFAULT_ENDPOINT = 'https://storage.scrapinghub.com/'
-    USERAGENT = 'python-hubstorage/{0}'.format(__version__)
+    DEFAULT_USER_AGENT = 'python-hubstorage/{0}'.format(__version__)
```

Comment:
Make sense, I didn't realise that keeping the attribute name is important but you're right, redefining it should be a common practice. Thanks!

@chekunkov  Have just fixed it, let me know if we miss anything else here.